Configs/.config/hyprdots/scripts/gamemode.sh

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env sh
 HYPRGAMEMODE=$(hyprctl getoption animations:enabled | sed -n '2p' | awk '{print $2}')
+
+# Waybar performance
+FILE="$HOME/.config/waybar/style.css"
```

Comment:
please note this file will be reset when you change waybar mode, so you have to enable and then disable gamemode again if you change the mode